session/session.go

```diff
@@ -312,9 +313,9 @@ func sanitizeBalInfo(balInfo perun.BalInfo) {
 	}
 }
 
-// retrieveParts retrieves the peers from corresponding to the aliases from the contacts provider.
+// retrieveParts retrieves the peers from corresponding to the aliases from the idprovider.
 // The order of entries for parts list will be same as that of aliases. i.e aliases[i] = parts[i].Alias.
-func retrieveParts(aliases []string, contacts perun.ContactsReader) ([]perun.Peer, error) {
+func retrieveParts(aliases []string, idProvider perun.IDProviderReader) ([]perun.Peer, error) {
```

Comment:
`ContactsReader` should be renamed to `PeerIDReader` instead of `IDProviderReader`.